test/Microsoft.DotNet.ToolPackage.Tests/SampleGlobalTool/consoledemo.csproj

```diff
@@ -7,5 +7,8 @@
       $(RestoreSources);
       https://api.nuget.org/v3/index.json;
      </RestoreSources>
+     <EnableSourceControlManagerQueries>false</EnableSourceControlManagerQueries>
```

Comment:
I think ideally these projects would be TestAssets which would be copied to the test execution folder and built during the test, instead of opting out of various arcade functionality.

Reply:
Agreed. My plan is to do that when I move to no longer handle test assets as part of the build.

Reply:
no, test assets require "preparetests" step and cannot work with straight up `dotnet test` and test explore. This is broken, it slow down dev cycle from seconds to minutes

Reply:
The plan is to not require any build steps and instead, for the tests themselves to prepare whatever they need. We can't keep doing things this way where part of it is done in one way and others in another, because we want some of it to be better. We should make all of it better.